key_list.go

```diff
@@ -73,7 +78,7 @@ func (kl *KeyList) AddAllPublicKeys(keys []PublicKey) *KeyList {
 }
 
 // String returns a string representation of the KeyList
```

Comment:
nit: Guess this is some old typo.

Reply:
I don't see typo in the comment

Reply:
Not a typo in the traditional sense but is String returns a string in the comment correct here?

Reply:
I think `String` is referring to the name of the method